ts/components/ui/IOScrollViewWithLargeHeader.tsx

```diff
@@ -117,17 +117,20 @@ export const IOScrollViewWithLargeHeader = forwardRef<View, Props>(
 
         {description && (
           <ContentWrapper>
-            <VSpacer size={4} />
+            <VSpacer size={16} />
             {typeof description === "string" ? (
               <Body color={theme["textBody-tertiary"]}>{description}</Body>
             ) : (
               <ComposedBodyFromArray body={description} textAlign="left" />
             )}
           </ContentWrapper>
         )}
-        <VSpacer size={16} />
-
-        {children}
```

Comment:
@ChrisMattew this change'll have effect on the whole app. Did we double check the team about this modification? cc @dmnplb

Reply:
We've encountered this graphical misalignment between the component's `spacing` and the one in the designs multiple times. After discussing it with @dmnplb, I decided to make the change. I haven't informed the rest of the team yet, but I think it's the right thing to let them know. Thanks!

Reply:
Good to go then! And good catch!